graph/community/bisect_test.go

```diff
@@ -137,6 +137,7 @@ func ExampleProfile_multiplex() {
 }
 
 func TestProfileUndirected(t *testing.T) {
+	t.Parallel()
```

Comment:
These tests are not safe here. You can file an issue for me to look into this later, but for the moment they should probably be backed out.

Reply:
Yup, I'll remove that commit.

I'm also wondering whether to keep or remove changes in packages that don't have significant impact?